### PR TITLE
untested task table style v centering status

### PR DESCRIPTION
**Pull request by stacyk**
## Link to Trello card (if applicable)
https://trello.com/c/kv3IuQwW/112-i-want-to-see-the-status-of-a-task

## Impacted areas of website
Project Page - Task Status Table, in the Status column

### CODE REVIEW

**Comment by stacyk** [comment]
[I want to see the status of a task.](https://trello.com/c/kv3IuQwW/112-i-want-to-see-the-status-of-a-task)

File: src/sass/components/_task-table.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Task Tables
+// ===========
+
+.project-task-status {
+  align-items: center;
+  display: flex;
```
Comment (jgerigmeyer) [inline src/sass/components/_task-table.scss]: This does not vertically center the text, and `display: flex` seems to remove table row borders.

Screenshot 2019-12-21 15 27 01

Comment (stacyk) [reply src/sass/components/_task-table.scss]: I couldn't test it locally, Im *guessing* there is a div inside the td perhaps? Whatever the structure, we will need the parent container set to display: flex; 

If I get this running locally I can test it out more

Comment (jgerigmeyer) [reply src/sass/components/_task-table.scss]: > If I get this running locally I can test it out more

Yeah, it's worth getting it running for you locally again. Has @wlonk already walked you through blowing up your DB and starting from scratch? It's relatively painless on this project.

Comment (wlonk) [reply src/sass/components/_task-table.scss]: (Just lemme know if you want/need help on that, @stacyk.)

File: src/sass/components/_task-table.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Task Tables
+// ===========
+
+// add a tiny top margin to create visual appearance of centered text
```
Comment (mirisuzanne) [inline src/sass/components/_task-table.scss]: Copy-paste error?

File: src/sass/components/_task-table.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+.project-task-status {
+  > div {
```
Comment (mirisuzanne) [inline src/sass/components/_task-table.scss]: The div shows an empty class attribute in dev tools, is there a way to pass it a class? This works fine but doesn't feel totally reliable. Even `div[title]` might be a bit more specific to the icon?

Comment (jgerigmeyer) [reply src/sass/components/_task-table.scss]: There is not a way to pass that div a class.

**Pull request merged**